Makefile

```diff
@@ -35,7 +35,7 @@ vet:
 	@echo ">> vetting code"
 	@$(GO) vet $(pkgs)
 
-megacheck:
+megacheck: 
```

Comment:
Any particular reason for this space?

Reply:
I think that was an accident, I'll change it